velox/expression/ExprToSubfieldFilter.cpp

```diff
@@ -536,8 +536,16 @@ std::pair<common::Subfield, std::unique_ptr<common::Filter>> toSubfieldFilter(
   if (auto call = asCall(expr.get())) {
     if (call->name() == "or") {
```

Comment:
I wonder if we should remove the logic for handling "or"... I don't think Axiom expects that logic.